owsi-core/owsi-core-components/owsi-core-component-jpa/src/main/java/fr/openwide/core/jpa/batch/processor/ThreadedProcessor.java

```diff
@@ -217,10 +218,10 @@ public void log(boolean force) {
 				if (StringUtils.hasText(loggerContext)) {
 					sb.append(loggerContext).append(" - ");
 				}
-				sb.append("Avancement {} / {} ({} ignorés, {} items / s. depuis début, {} items / s. depuis dernier log)");
+				sb.append("In progress: {} / {} ({} ignored, {} items/s since start, {} items/s since last log)");
 				
 				if (propertyService.get(MIGRATION_LOGGING_MEMORY)) {
```

Comment:
This dependency to a spring service in this very low-level class disturbs me a bit.
We could avoid using the propertyService inside the ThreadedProcessor entirely if we simply used two loggers: one with the memory trace, the other without. With a few `logger.isInfoEnabled()`, we wouldn't even lose in performance.
@lalmeras , do you think this solution would fit your needs? You would then have to enable one logger or the other, depending on whether you want a memory trace or not.